app/cli/internal/action/attestation_status.go

```diff
@@ -150,6 +138,61 @@ func (action *AttestationStatus) Run(ctx context.Context, attestationID string)
 	return res, nil
 }
 
+// populateContractMaterials populates the materials that are defined in the contract schema
+func (action *AttestationStatus) populateContractMaterials(res *AttestationStatusResult, visitedMaterials map[string]struct{}) error {
+	for _, m := range action.c.CraftingState.InputSchema.Materials {
+		materialResult := &AttestationStatusResultMaterial{
+			Material: &Material{
+				Name: m.Name, Type: m.Type.String(),
+				Annotations: pbAnnotationsToAction(m.Annotations),
+			},
+			IsOutput: m.Output, Required: !m.Optional,
+		}
+
+		if cm, found := action.c.CraftingState.Attestation.Materials[m.Name]; found {
+			if err := setMaterialValue(cm, materialResult.Material); err != nil {
+				return err
+			}
+			materialResult.Set = true
```

Comment:
it might make sense to add these two lines (set and tag) inside `setMaterialValue`

Reply:
Sure